src/components/activityEventElems/ActivityElement/ActivityElement.tsx

```diff
@@ -9,16 +10,36 @@ import { contentLineHeight, smallHeaderStyle } from '../styles'
 
 import { ActivityElementEvent } from './activityElementEvent'
 
-const DetailsContainer: React.FC = ({ children }) => {
-  return (
+const CallerBeneficiary = ({
+  caller,
+  beneficiary,
+}: {
+  caller?: string
+  beneficiary?: string
+}) => {
+  if (!beneficiary && !caller) return null
+
+  return beneficiary &&
+    caller &&
+    beneficiary.toLowerCase() !== caller.toLowerCase() ? (
     <div
       style={{
-        display: 'flex',
-        justifyContent: 'space-between',
-        alignContent: 'space-between',
+        lineHeight: contentLineHeight,
```

Comment:
Any chance we can get rid of this?

Reply:
actuallyyy i think with the change i just made this is unnecessary now. will verify

Reply:
done